Tests/SentryTests/ViewCapture/SentryUIRedactBuilderTests+SpecialViews.swift

```diff
@@ -0,0 +1,404 @@
+#if os(iOS) && !targetEnvironment(macCatalyst)
+import AVKit
+import Foundation
+import PDFKit
+import SafariServices
+@_spi(Private) @testable import Sentry
+import SentryTestUtils
+import SnapshotTesting
+import SwiftUI
+import UIKit
+import WebKit
+import XCTest
+
+// The following command was used to derive the view hierarchy:
+//
+// ```
+// (lldb) po rootView.value(forKey: "recursiveDescription")!
+// ```
+class SentryUIRedactBuilderTests_SpecialViews: SentryUIRedactBuilderTests { // swiftlint:disable:this type_name
+    private var rootView: UIView!
+
+    private func getSut(maskAllText: Bool, maskAllImages: Bool) -> SentryUIRedactBuilder {
+        return SentryUIRedactBuilder(options: TestRedactOptions(
+            maskAllText: maskAllText,
+            maskAllImages: maskAllImages
+        ))
+    }
+
+    override func setUp() {
+        rootView = UIView(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+    }
```

Comment:
`l`: I like to avoid global test variables to minimize side effects or shared state for the tests. I know we have them all over our test code, but I don't really like it. What about having a helper function `getRootView`, removing the `rootView` and let every test have its own local variable of `rootView`?